web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -58,14 +63,18 @@
                     <div id="stream_settings_title" class="stream-info-title">{{t 'Channel settings' }}</div>
                 </div>
                 <div class="nothing-selected">
+                    <button type="button" class="create_stream_button animated-purple-button" {{#unless can_create_streams}}disabled{{/unless}}>{{t 'Create channel' }}</button>
                     {{#if can_create_streams}}
-                    <button type="button" class="create_stream_button animated-purple-button">{{t 'Create channel' }}</button>
```

Comment:
instead of adding class `disabled`, what we do in other places is add the attribute disabled="true". No additional CSS is required.

Reply:
I think you are mistaken it is not a class `disabled`  is a attribute.

Reply:
Oh, the CSS confused me. Why is additional CSS required then?